core/whistle_apps-1.0.0/src/whapps_speech.erl

```diff
@@ -290,6 +354,37 @@ create_response(_Engine, {'error', _R}) ->
 create_response(_Engine, {'ibrowse_req_id', ReqID}) ->
     lager:debug("speech file streaming as ~p", [ReqID]),
     {'ok', ReqID};
+create_response(<<"voicefabric">> = _Engine, {'ok', "200", Headers, Content}) ->
+    [lager:debug("hdr: ~p", [H]) || H <- Headers],
```

Comment:
@jamesaimonetti 
Without headers it will be very hard to resolve problems with this service.
It doesn't have format option, and it may change raw to something else by default in future.
Also it may deliver media in one response or by chunks.
Long story short - we guarantee these logs are needed when you deal with voicefabric :)